### PR TITLE
MAINT-28042 : Improve performance when checking if user is a redactor in a space

### PR DESCRIPTION
**Pull request by ahamdi**
We improved the performance of the call that checks if a user is redactor in a space. The latency caused that the button is displayed a while after opening the composer in case of spaces with many members (1000 members)
The Post news page also displayed an error for a while (15s) before displaying the input fields which is fixed also with this PR

### CODE REVIEW

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java [hunk 1]
```diff
@@ -1195,18 +1198,8 @@ public void unarchiveNews(String newsId) throws Exception {
...
-    String spaceGroupId = space.getGroupId();
-    
-    List<String> spaceRedactors = SpaceUtils.findMembershipUsersByGroupAndTypes(spaceGroupId, "redactor");
-    if (!spaceRedactors.isEmpty()) {
-      spaceHasRedactors = true;
-      //check if connected user is redactor
-      isCurrentUserRedactor = spaceRedactors.contains(authenticatedUser);
-    }
-    return (!spaceHasRedactors || isCurrentUserRedactor);
+  public boolean canCreateNews(String authenticatedUser, Space space) throws Exception {
```
Comment (azayati) [inline services/src/main/java/org/exoplatform/news/NewsServiceImpl.java:1201]: IMO, this method is not needed anymore if it is called only here

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java (hunk 1 above)
Comment (ahamdi) [reply services/src/main/java/org/exoplatform/news/NewsServiceImpl.java:1201]: It is !

Comment (ahamdi) [reply services/src/main/java/org/exoplatform/news/NewsServiceImpl.java]: Done

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**